src/runners/handlers/slack.py

```diff
@@ -65,37 +65,61 @@ def handle(
 ):
     slack_token_ct = slack_api_token or api_token
     slack_token = vault.decrypt_if_encrypted(slack_token_ct)
-
     sc = SlackClient(slack_token)
 
     # otherwise we will retrieve email from assignee and use it to identify Slack user
     # Slack user id will be assigned as a channel
 
-    title = alert['TITLE']
-
+    # title = alert['TITLE']
+    title = 'title'
     if recipient_email is not None:
-        result = sc.api_call("users.lookupByEmail", email=recipient_email)
-
-        # log.info(f'Slack user info for {email}', result)
-
-        if result['ok'] is True and 'error' not in result:
-            user = result['user']
-            userid = user['id']
-        else:
-            raise RuntimeError(f'no Slack user for email {recipient_email}')
-
+        if isinstance(recipient_email, str):
+                user = sc.api_call("users.lookupByEmail", email=recipient_email)
+                print(user)
+                if not user['ok']:
+                    log.error(f'Cannot identify Slack user for email {recipient_email}')
```

Comment:
if there's an error, you want to raise it rather than returning None (which signals everything was Ok)